boards/arm/lpcxpresso55s28/lpcxpresso55s28_ns.dts

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (c) 2020 Lemonbeat GmbH
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <nxp/nxp_lpc55S2x_ns.dtsi>
+#include "lpcxpresso55s28_common.dtsi"
+
+/ {
+	model = "NXP LPCXpresso55S28 board (non-secure)";
+	compatible = "nxp,lpc55xxx", "nxp,lpc";
+
+	aliases {
+		sw0 = &user_button_1;
+		sw1 = &user_button_2;
+		sw2 = &user_button_3;
+	};
+
+	chosen {
+		zephyr,sram = &non_secure_ram;
+		zephyr,flash = &flash0;
+		zephyr,code-partition = &slot1_partition;
```

Comment:
I am not sure how is is supposed to work. How can i test the board lpcxpresso55s28_ns without mcuboot and trusted environment?